packages/browser-integration-tests/suites/public-api/startTransaction/init.js

```diff
@@ -1,6 +1,5 @@
 import * as Sentry from '@sentry/browser';
-// eslint-disable-next-line no-unused-vars
-import * as _ from '@sentry/tracing';
```

Comment:
I think we did this on purpose, not sure if the change is safe

Reply:
The change should be safe, I validated by running integration tests.

Reply:
nvm it isn't I reverted.